slices/nginx.yaml

```diff
@@ -14,6 +14,7 @@ slices:
       - libpcre2-8-0_libs
       - libssl3t64_libs
       - nginx-common_config
+      - nginx-common_modules
```

Comment:
Without this change, users would be required to add `nginx-common_modules` manually in order to use NGINX modules out-of-the-box.
When this slice is missing, the following error occurs:
```shell
2024/12/17 09:05:31 [emerg] 233876#233876: dlopen() "/usr/share/nginx/modules/ngx_http_headers_more_filter_module.so" failed (/usr/share/nginx/modules/ngx_http_headers_more_filter_module.so: cannot open shared object file: No such file or directory) in /etc/nginx/modules-enabled/50-mod-http-headers-more-filter.conf:1
nginx: configuration file /etc/nginx/nginx.conf test failed
```

Reply:
Interesting, this is a broken symlink ...but it appears to be that way intentionally. 

It does make sense to have it and I confirm the test doesn't work without it